Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "¤" : " "); // mark done task with X
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void markAsUndone() {
+        this.isDone = false;
+    }
+
+    public String toString() {
+        if (isDone){
+            return ("[T]" + "[" + this.getStatusIcon() + "] " + this.description);
+        }
+        else {
+            return ("[T]" + "[ ] " + this.description);
+        }
```

Comment:
According to the coding standard document shared in class, the 'else' should be put beside the closing bracket of 'if':
```
if (condition) {
    statements;
} else {
    statements;
}
```